scripts/ci-develop.sh

```diff
@@ -22,16 +22,14 @@ DOCROOT="web"
 cd ..
 
 # Create a new Drupal project.
-composer create-project drupal/recommended-project:9.1.9 $APP_NAME --no-install --no-interaction
+composer create-project drupal/recommended-project:9.2.2 $APP_NAME --no-install --no-interaction
 
 # Move into the directory.
 cd $APP_NAME;
 
 # Change the minimum stability flag.
 $COMPOSER config minimum-stability "dev"
 
-$COMPOSER require "zaporylie/composer-drupal-optimizations:^1.1.2" --no-update
```

Comment:
Did you mean to remove this?

Reply:
yes, that is for composer 1 only, it's not needed for composer 2